src/state/slices/opportunitiesSlice/selectors/stakingSelectors.ts

```diff
@@ -466,6 +466,7 @@ export const selectEarnBalancesFiatAmountFull = createDeepEqualOutputSelector(
     aggregatedUserStakingOpportunities
       .map(opportunity => makeOpportunityTotalFiatBalance({ opportunity, marketData, assets }))
       .reduce((acc, opportunityFiatAmount) => acc.plus(opportunityFiatAmount), bn(0))
+      // TODO(gomes): add all UNI/V2 asset balances here
```

Comment:
Looks straightforward but isn't, will need some rearchitect in a follow-up to get around the circular dep error from declaring this guy

```typescript
const { getOpportunityIds } = opportunitiesApi.endpoints
const selectUniV2LpIds = getOpportunityIds.select({
  defiType: DefiType.LiquidityPool,
  defiProvider: DefiProvider.UniV2,
})
```

Since `opportunitiesApi` is also imported in UNI-V2 resolver to `selectGetZapperAppBalancesOutput(state)`